feature-libs/product/multi-dimensional/root/variants-multi-dimensional-root.module.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { NgModule } from '@angular/core';
+import { provideDefaultConfigFactory } from '@spartacus/core';
+import { PRODUCT_MULTIDIMENSIONAL_FEATURE } from './feature-name';
+
+export function defaultProductMultiDimensionalComponentsConfig() {
+  return {
+    featureModules: {
+      [PRODUCT_MULTIDIMENSIONAL_FEATURE]: {
+        cmsComponents: [
+          'ProductVariantSelectorComponent', // TBD trigger
```

Comment:
[MEDIUM]
As we discussed in call, let's assign a different component type to this library as a Lazy Loading trigger, to allow customers to possibly use any of those options:
(a) only simple Variant products
(b) only Multidimensional products
(c) some simple Variant products and some Multidimensional

Reply:
I'll do it a separate sub-ticket